### PR TITLE
[Mellanox][201911] Change buffer config for new SKU Mellanox-SN2700-D40C8S8

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Why I did it

Change buffer config for new SKU Mellanox-SN2700-D40C8S8

#### How I did it

Reuse the buffer config of SKU Mellanox-SN2700-D48C8

#### How to verify it

Run sonic-mgmt qos test and all passed

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012

### CODE REVIEW

**Review by stephenxs — APPROVED** [review]

File: device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D40C8S8/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -1,102 +0,0 @@
-{% set default_cable = '5m' %}
-{% set ingress_lossless_pool_size =  '7719936' %}
-{% set ingress_lossless_pool_xoff =  '1032192' %}
-{% set egress_lossless_pool_size =  '13945824' %}
-{% set egress_lossy_pool_size =  '7719936' %}
-
-{%- macro generate_port_lists(PORT_ALL) %}
-    {# Generate list of ports #}
-    {%- for port_idx in range(0, 32) %}
-        {%- if PORT_ALL.append("Ethernet%d" % (port_idx)) %}{%- endif %}
```
Comment (neethajohn) [inline device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D40C8S8/buffers_defaults_t0.j2:10]: Wouldn't the indexing be different for both the sku's?

**Comment by qiluo-msft** [comment]
Please submit a master PR first, and request cherry-pick to other branches. If clean cherry-pick is impossible, we can keep review this PR.

**Review by qiluo-msft — CHANGES_REQUESTED** [review]
block

File: device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D40C8S8/buffers_defaults_t0.j2 (hunk 1 above)
Comment (stephenxs) [reply device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D40C8S8/buffers_defaults_t0.j2:10]: Looks like this code is not used. I suggest removing it in aother PR

**Comment by Junchao-Mellanox** [comment]
Closed, as there are PRs in master and we will do cherry-pick once the master PR get merged.

**Pull request closed without merging**